client/my-sites/customer-home/cards/notices/reader-first-posts/style.scss

```
@@ -1,4 +1,4 @@
-.reader-first-posts__nudge {
+.card.reader-first-posts__nudge {
```

```suggestion
.reader-first-posts__nudge,
.card.reader-first-posts__nudge {
```
I think it might be useful to have the "generalized" version of this rule.
At the same time, it is not used outside a card.
So, this suggestion is highly optional.

---

Thanks. Let's go with just the scoped one for now